src/components/App.js

```diff
@@ -1,12 +1,15 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { inputChange, setSearchKeyword, getCategories, setAddresses, selectAddress, saveAddress, addressInputChange, showAddressWindow, hideAddressWindow } from './../redux/actions';
+import { inputChange, setSearchKeyword, getCategories, setAddresses, selectAddress, saveAddress,
+  addressInputChange, showAddressWindow, hideAddressWindow, getGeolocation } from './../redux/actions';
+import './../redux/actions';
```

Comment:
I split up this long import line into two because it was getting hard to read such a long line. Anyway, if this goes against our style, I can revert it!